data/countries/SE.yaml

```diff
@@ -44,41 +80,12 @@ holidays:
           sv: Midsommardagen
           en: Midsummer Day
           fi: Juhannuspäivä
-      saturday after 10-31:
-        name:
-          sv: Alla Helgons dag
-      12-25:
-        _name: 12-25
-      12-26:
-        _name: 12-26
-
-      01-13:
+      friday after 10-30:
         name:
-          sv: Tjugondag Knut
-        type: observance
-      sunday in March:
-        name:
-          sv: Vasaloppet
-        type: observance
-      03-25:
-        name:
-          sv: Marie Bebådelsedag
-          lat: Annuntiatio Mariæ
-        type: observance
-      04-30:
-        name:
-          sv: Valborgsmässoafton
-          en: Walpurgis Night
-        type: observance
-      easter:
-        _name: easter
-        type: observance
-      easter 49:
-        _name: easter 49
-        type: public
-      easter 50:
-        _name: easter 50
-        type: observance
+          sv: Allhelgonaafton
```

Comment:
Sorry for catching this a bit late, but this isn’t a public holiday. Just an observance